external/storm-kafka-client/README.md

```diff
@@ -1,192 +1,5 @@
-#Storm Kafka Spout with New Kafka Consumer API
+#Storm Kafka integration using the kafka-client jar
```

Comment:
can we call it as Kafka New consumer API. Not many understand its using newer api by calling it as "kafka-client" jar